tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": ["es2017", "dom"],
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "strict": true,
+    "noImplicitThis": true,
+    "alwaysStrict": true,
+    "types": ["node", "mocha"],
+    "esModuleInterop": true
+  },
+  "include": ["packages/core/**/*.js", "packages/tabs/**/*.js"],
+  "exclude": [
+    "node_modules",
+    "**/node_modules/*",
+    "**/coverage/*",
+    "**/dist/**/*",
+    "packages/**/demo/**/*",
+    "packages/**/test",
```

Comment:
Do we wanna type lint test files? I saw Thomas did add JSDocs types for some tests.